packages/compass-components/package.json

```diff
@@ -71,6 +71,7 @@
     "@leafygreen-ui/tokens": "^1.4.1",
     "@leafygreen-ui/tooltip": "^9.0.2",
     "@leafygreen-ui/typography": "^15.2.0",
+    "@mongodb-js/compass-logging": "^1.1.6",
```

Comment:
`compass-logging` creates a dependency on electron and node APIs in a package that we decided should stay unaware of compass internals

Reply:
that's a good point. having GC tracking outside of this component will make it cumbersome to track when using the component. 
I'll create a follow up ticket as suggested to remove this dependency.

Reply:
FWIW this is the way to go about this with the tracking interface we have and we will be doing this for signals as tracking should stay close to the actual tracked use-case instead of being a generic catch-all, otherwise we should've designed the tracking system with a catch-all interface in the first place (it was like that before and it was hard to reason about as this was creating implicit dependencies all over the place)

Reply:
Removed dependency and added Provider to accomplish this.